ExeterTeam25/group-software-project
# Interactive map rewrite

To test user location finding and map:

- Run the server
- Open home directory
- Approve geolocation access
- Check that marker is close to your current location (accuracy is spotty without GPS)

These steps will only work on browsers that accept JavaScript geolocation APIs on HTTP connections. May not work on android; should implement HTTPS to improve accuracy

To test challenge marker placing:

- Create a user with challenge creation privileges
- Create a challenge in a location within the map's bounds (latLong.json has some good examples)
- Open home directory and click on markers verify that they are in their appropriate locations

Details:

Removed folium map functionality in views.py
Implemented Leaflet map in home.html template with marker placement and user location tracking
Began implementing methods for handling user location data
Updated README to reflect changes
Updated views to reflect these changes
